float64 pwm_rudder
float64 pwm_main_sail
float64 pwm_fore_sail
